### PR TITLE
re adds back the WFR

### PR DESCRIPTION
**Pull request by thedarkcrusader**
## About The Pull Request
Aadds back in the WFR

## Why It's Good For The Game
it was supposed to be in removed by mistake

## Testing
 it works

## Changelog
:cl:
add: Added the WPR to the serb crate adds boxes which can be found in maint aswell
/:cl:

### CODE REVIEW

File: code/modules/projectiles/ammunition/bullets.dm [hunk 1]
```diff
@@ -476,6 +488,12 @@
 	matter = list(MATERIAL_STEEL = 3, MATERIAL_PLASTEEL = 3)
 	rarity_value = 25
 
+/obj/item/ammo_casing/grenade/frag/phosphorus
+	name = "SA WFR \"Micic\""
+	desc = "A military-grade White phosphorus grenade shell, designed to be fired from grenade launchers. The abbreviation \"SA\" marks its producer, but the lack of detatil makes it ubiquitous."
+	icon_state = "high_incendiary
```
Comment (Firefox13) [inline code/modules/projectiles/ammunition/bullets.dm:494]:
```suggestion
	icon_state = "high_incendiary"
```
You definitely didn't test this.

**Comment by Firefox13** [comment]
You're conflincting with your own PR...

**Comment by thedarkcrusader** [comment]
> You're conflincting with your own PR...

Your killing me here

**Comment by thedarkcrusader** [comment]
> > You're conflincting with your own PR...
> 
> Your killing me here

how does  that even happen

**Comment by Firefox13** [comment]
Instead of adding back just WPR you copied all the changes

**Comment by thedarkcrusader** [comment]
> > > You're conflincting with your own PR...
> > 
> > 
> > Your killing me here
> 
> how does that even happen

maybe later im going to go on vacation

**Pull request closed without merging**